### PR TITLE
Add Kubernetes on GCP GKE Hello World Tutorial

### PR DESCRIPTION
**Pull request by metral**

### CODE REVIEW

File: _includes/gcp-gke-prereqs.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## Prerequisites
+
+1. [Install Pulumi](https://pulumi.io/install)
+1. [Install Node.js 8.11.3](https://nodejs.org/en/download/)
```
Comment (lblackstone) [inline _includes/gcp-gke-prereqs.md]: We do support Node 6, but this should work too

File: quickstart/gcp/tutorial-gke.md [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+		namespaceName  : "helloworld-pz4u5kyq"
+		serviceName    : "helloworld-l61b5dby"
+		servicePublicIP: "35.236.26.151"
+
+	Resources:
+		+ 6 created
+
+	Duration: 3m51s
+
+	Permalink: https://app.pulumi.com/metral/gcp-ts-gke-hello-world/gke-demo/updates/1
```
Comment (lblackstone) [inline quickstart/gcp/tutorial-gke.md]: Might want to anonymize this link

File: quickstart/gcp/tutorial-gke.md [hunk 3]
```diff
@@ -0,0 +1,380 @@
...
+
+
+## Access the Kubernetes Cluster using Pulumi Providers
+
+Now that we have an instance of Kubernetes running, we may want to create API resources in Kubernetes to manage our workloads through Pulumi.
+We can do this by configuring a Pulumi provider to our newly created cluster, and instantiating a new Kubernetes resource object in our Pulumi program.
```
Comment (lblackstone) [inline quickstart/gcp/tutorial-gke.md]:
```suggestion
We can do this by configuring a Pulumi provider for our newly created cluster, and instantiating a new Kubernetes resource object in our Pulumi program.
```

File: quickstart/gcp/tutorial-gke.md [hunk 4]
```diff
@@ -0,0 +1,380 @@
...
+$ kubectl create deployment my-nginx --image=nginx
+$ kubectl get pods
+$ kubectl delete deployment my-nginx
+```
+
+## Experimentation
+
+From here on, feel free to experiment. Simply making edits and running `pulumi up` afterwords, will incrementally update your stack.
+
+For example, if you wish to pull in existing Kubernetes YAML manifests into
```
Comment (lblackstone) [inline quickstart/gcp/tutorial-gke.md]:
```suggestion
For example, if you wish to pull existing Kubernetes YAML manifests into
```

File: quickstart/gcp/tutorial-gke.md [hunk 5]
```diff
@@ -0,0 +1,380 @@
...
+Pulumi to aid in your transition, append the following code block to the existing
+`index.ts` file and run `pulumi up`.
+
+This is an example of how to create the standard Kubernetes Guestbook manifests in
+Pulumi using the Guestbook its YAML manifests. We take the additional steps of transforming
```
Comment (lblackstone) [inline quickstart/gcp/tutorial-gke.md]:
```suggestion
Pulumi using the Guestbook YAML manifests. We take the additional steps of transforming
```

**Comment by metral** [comment]
Check is currently failing as its dependent on https://github.com/pulumi/examples/pull/226, which is still being checked by TravisCI. will re-run once that PR lands

**Pull request merged**